components/view_image/view_image.tsx

```diff
@@ -208,7 +244,10 @@ export default class ViewImageModal extends React.PureComponent {
     handleGetPublicLink = () => {
         this.handleModalClose();
 
-        GlobalActions.showGetPublicLinkModal(this.props.fileInfos[this.state.imageIndex].id);
+        const imageIndex = this.state.imageIndex ?? 0;
+        const fileInfos = this.props?.fileInfos ?? [];
```

Comment:
```suggestion
        const fileInfos = this.props.fileInfos ?? [];
```
I don't think props can be undefined